src/components/CryptoDetail.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
```

Comment:
- [ ] Please fix this console error when trying to test your project
![Screenshot from 2021-11-20 18-38-29](https://user-images.githubusercontent.com/62705947/142736001-4e3768c3-bc04-4ccc-a032-0a9c13d2bf99.png)